Update words.md - stilistische Änderungsvorshläge L76 - L98

TQueV opened the pull request.

TQueV committed:
Update words.md - stilistische Änderungsvorshläge L76 - L98

herrhundt commented on words/words.md:
```diff
@@ -73,11 +73,11 @@ Wenn wir "eine Verdoppelung alle 4 Tage", bei einer Bevölkerung, die mit nur 0,
 </div>
 
-Dies ist die **exponentielle Wachstumskurve.** Sie fängt klein an und explodiert dann. "Oh, es ist nur eine Grippe" bis "Oh richtig, Grippe schafft keine *Massengräber in reichen Städten*". 
+Dies ist die **exponentielle Wachstumskurve.** Sie fängt klein an und explodiert dann. Von "Ach, es ist nur eine Grippe" bis "Oh richtig, eine Grippe schafft keine *Massengräber in reichen Städten*". 
```
Klingt eindeutig besser.

jkoelling replied:
Würde 
"Oh richtig, eine Grippe schafft keine *Massengräber in reichen Städten*""
evtl. ersetzen durch 
"Oh richtig, eine Grippe schafft legt keine *Massengräber in reichen Städten* an"

"Massengräber schaffen" klang für mich etwas seltsam. "Massengräber anlegen" scheint nach kurzer Recherche gängig. Einfach übernehmen wenn es sich gut anfühlt, sonst ist egal.

jkoelling replied:
find's so auch gut

jkoelling commented on words/words.md:
```diff
@@ -93,9 +93,9 @@ Dies ist eine "S-förmige" **logistische Wachstumskurve.** Sie fängt klein an,
 Aber auch diese Simulation ist *immer noch* falsch. Wir übersehen die Tatsache, dass <icon i></icon> ansteckenden Menschen schließlich aufhören, ansteckend zu sein, entweder indem sie 1) sich erholen, 2) sich mit Lungenschäden "erholen" oder 3) sterben.
 
-Der Einfachheit halber wollen wir so tun, als ob alle <icon i></icon> ansteckenden Menschen wieder genesen <icon r></icon> würden. (Denken Sie aber daran, dass in Wirklichkeit einige von ihnen sterben.) <icon r></icon> nicht wieder infiziert werden können, und lassen Sie uns - *vorläufig!* - so tun, als ob sie ein Leben lang immun bleiben.
+Der Einfachheit halber wollen wir so tun, als ob alle <icon i></icon> ansteckenden Menschen wieder genesen <icon r></icon> würden. (Denkt aber daran, dass in Wirklichkeit einige von ihnen sterben.) <icon r></icon> können nicht wieder infiziert werden, lasst ns - *vorläufig!* - so tun, als ob sie ein Leben lang immun bleiben.
```
```suggestion
Der Einfachheit halber wollen wir so tun, als ob alle <icon i></icon> ansteckenden Menschen wieder genesen <icon r></icon>. (Denkt aber daran, dass in Wirklichkeit einige von ihnen sterben.) <icon r></icon> können nicht wieder infiziert werden, lasst ns - *vorläufig!* - so tun, als ob sie ein Leben lang immun bleiben.
```
finde das "würden" hier unnötig, und wenn wir es behalten würden, müssten wir den Rest auch entsprechend umschreiben (zB "sterben würden")

jkoelling replied:
```suggestion
Der Einfachheit halber wollen wir so tun, als ob alle <icon i></icon> ansteckenden Menschen wieder genesen <icon r></icon> würden. (Denkt aber daran, dass in Wirklichkeit einige von ihnen sterben.) <icon r></icon> können nicht wieder infiziert werden, lasst uns - *vorläufig!* - so tun, als ob sie ein Leben lang immun bleiben.
```

jkoelling commented on words/words.md:
```diff
@@ -93,9 +93,9 @@ Dies ist eine "S-förmige" **logistische Wachstumskurve.** Sie fängt klein an,
+Der Einfachheit halber wollen wir so tun, als ob alle <icon i></icon> ansteckenden Menschen wieder genesen <icon r></icon> würden. (Denkt aber daran, dass in Wirklichkeit einige von ihnen sterben.) <icon r></icon> können nicht wieder infiziert werden, lasst ns - *vorläufig!* - so tun, als ob sie ein Leben lang immun bleiben.
 
-Mit COVID-19 sind Sie *im Durchschnitt* schätzungsweise 10 Tage lang  ansteckend.[^infectiousness] Das bedeutet, dass manche <icon i></icon> Menschen sich früher als nach 10 Tagen erholen, andere später. **So sieht das bei einer Simulation aus, die mit 100 % <icon i></icon> *beginnt*:**
+Es wird angenommen, dass sie mit COVID-19 *im Durchschnitt* 10 Tage lang ansteckend sind.[^infectiousness] Das bedeutet, dass manche <icon i></icon> Menschen sich früher als nach 10 Tagen erholen, andere später. **So sieht das bei einer Simulation aus, die mit 100 % <icon i></icon> *beginnt*:**
```
```suggestion
Es wird angenommen, dass Personen mit COVID-19 *im Durchschnitt* 10 Tage lang ansteckend sind.[^infectiousness] Das bedeutet, dass manche <icon i></icon> Menschen sich früher als nach 10 Tagen erholen, andere später. **So sieht das bei einer Simulation aus, die mit 100 % <icon i></icon> *beginnt*:**
```
"Sie" wollten wir grundsätzlich mit "du" ersetzen. 
Das "you" im Original hier ist eher als "man" gemeint.
Hab "Personen" statt "man" bevorzugt.

TQueV committed:
Update words/words.md

Co-authored-by: Jan Kölling <jkoelling@users.noreply.github.com>

TQueV committed:
Update words/words.md

Ich glaub das war auch eher ein Plural-"Sie", so oder so, "Personen" ist besser

Co-authored-by: Jan Kölling <jkoelling@users.noreply.github.com>
